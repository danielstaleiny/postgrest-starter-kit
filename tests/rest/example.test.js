describe('my beverage', () => {
  test('is delicious', () => {
      expect({"hi": "there"}).toMatchSnapshot();
  });

  test('Dummy test test', () => {
      expect({"name": "Danie"}).toMatchSnapshot();
  });
});
