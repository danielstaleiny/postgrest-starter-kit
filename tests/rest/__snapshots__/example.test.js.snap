// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my beverage Dummy test test 1`] = `
Object {
  "name": "Danie",
}
`;

exports[`my beverage is delicious 1`] = `
Object {
  "hi": "there",
}
`;
